test/Microsoft.AspNetCore.Antiforgery.Test/Internal/DefaultAntiforgeryTest.cs

```diff
@@ -276,6 +277,67 @@ public void GetAndStoreTokens_ExistingValidCookieToken_NotOverriden()
         }
 
         [Fact]
+        public void GetAndStoreTokens_ExistingValidCookieToken_NotOverriden_AndSetsDoNotCacheHeaders()
```

Comment:
The behavior here is similar to `SetCookieTokenAndHeader`. Not sure If I confused you with the method name here but cache headers are always overriden...the `NotOverriden` part here was for the cookie token.